chall01/hhuhtane.py

```diff
@@ -0,0 +1,37 @@
+# **************************************************************************** #
+#                                                                              #
+#                                                         :::      ::::::::    #
+#    hhuhtane.py                                        :+:      :+:    :+:    #
+#                                                     +:+ +:+         +:+      #
+#    By: hhuhtane <hhuhtane@student.hive.fi>        +#+  +:+       +#+         #
+#                                                 +#+#+#+#+#+   +#+            #
+#    Created: 2020/03/26 15:47:35 by hhuhtane          #+#    #+#              #
+#    Updated: 2020/03/26 16:51:09 by hhuhtane         ###   ########.fr        #
+#                                                                              #
+# **************************************************************************** #
+
+import sys
+
+alphabet = "abcdefghijklmnopqrstuvwxyz"
+capital = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
```

Comment:
you can use python built-in function for that:
```python
alphabet = string.ascii_lowercase
```

you did not need to have a capital array